CMake/utils/kwiver-utils-python.cmake

```diff
@@ -124,15 +124,12 @@ function (kwiver_add_python_module path     modpath    module)
     if (python_noarch)
       return ()
     else ()
-      set(python_install_path lib)
```

Comment:
Is this and the one below now empty "else" statements?

Reply:
Good point - fixed